pages/api/me.ts

```diff
@@ -0,0 +1,40 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import prisma from "@lib/prisma";
+import { getSession } from "@lib/auth";
+import { defaultAvatarSrc } from "@lib/profile";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const session = await getSession({ req: req });
+  if (!session) {
+    res.status(401).json({ message: "Not authenticated" });
+    return;
+  }
+
+  const user: User = await prisma.user.findUnique({
+    where: {
+      id: session.user.id,
+    },
+    select: {
+      id: true,
+      username: true,
+      name: true,
+      email: true,
+      bio: true,
+      timeZone: true,
+      weekStart: true,
+      startTime: true,
+      endTime: true,
+      bufferTime: true,
+      theme: true,
+      createdDate: true,
+      hideBranding: true,
+      avatar: true,
+    },
```

Comment:
Maybe you don't need all of them? :)

Reply:
It's an API endpoint; so I don't but when someone uses the API we can't tell what data would be useful or what they'd use to integrate :)